packages/core/src/components/Tipseen/Tipseen.tsx

```diff
@@ -210,7 +206,6 @@ const Tipseen: VibeComponent<TipseenProps> & {
 export default withStaticProps(Tipseen, {
   closeButtonThemes: TipseenCloseButtonTheme,
   animationTypes: AnimationType,
-  justifyTypes: JustifyType,
```

Comment:
I think we should document this one as well, we might not add this to the final migration guide since it can be covered with a codemod but I think we better keep it documented for us